package.json

```diff
@@ -19,7 +19,7 @@
     "build-awesome": "script/awesome",
     "build-userland": "script/userland",
     "release": "script/release 2>&1 | script/notify",
-    "dev": "nodemon -e js,scss,yml server.js",
+    "dev": "NODE_PATH=. nodemon -e js,scss,yml server.js",
```

Comment:
What is this `NODE_PATH` thing for?

Reply:
Add `NODE_PATH` we can quickly `require` anything in project with a `path` no need to think about  not absolute path. Like I can do here: [routes/userland/show.js#line6](https://github.com/electron/electron.atom.io/pull/799/files/3b56398c6cb4e84666a4c8e1da429ee44b8efe6b#diff-6606e56f8941159c4982e8790c9bcef0R6)